evo1/atb/__init__.py

```diff
@@ -1,11 +1,18 @@
-from evo1.atb.base import EncounterID
-from evo1.atb.farming import FarmingGoal, SeqATBmove2D, calc_next_encounter
+from evo1.atb.base import SeqATBCombat
+from evo1.atb.encounter import Encounter, EncounterID, calc_next_encounter
```

Comment:
I spent some time refactoring how ATB combat handled in the code, splitting stuff up into smaller files.